disnake/interactions/base.py

```diff
@@ -612,7 +612,7 @@ def is_done(self) -> bool:
         """
         return self._responded
 
-    async def defer(self, *, ephemeral: bool = False) -> None:
+    async def defer(self, *, show_message: bool = False, ephemeral: bool = False) -> None:
```

Comment:
I would like to have `show_message` after `ephemeral`, but it's just a small recommendation.

Reply:
I put it first because it made more sense in the docstring, IMO.

Reply:
What do you mean by "it made more sense in docstrings"? Is it because the docstring param is first?

Reply:
> What do you mean by "it made more sense in docstrings"? Is it because the docstring param is first?

Yeah, I wanted to reference the show_message param but don't want to backref, and mention a kwarg that's second. 

Its not *really* a problem whichever way they're defined as they're keyword only.

Reply:
https://github.com/DisnakeDev/disnake/pull/248/commits/fd377e894122e2d9f4f1a6f494ed6baf82ac086f